src/Sidebar.js

```diff
@@ -13,79 +11,69 @@ import AppsIcon from "@material-ui/icons/Apps";
 import ExpandLessIcon from "@material-ui/icons/ExpandLess";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import AddIcon from "@material-ui/icons/Add"
+import { collection, getDocs } from "firebase/firestore"
+
+import SidebarOptions from "./SidebarOptions"
 import db from "./firebase"
-import {collection , getDocs , doc} from "firebase/firestore/lite"
+import "./Sidebar.css"
 
 
 function Sidebar() {
-    const [channels, setChannels] = useState([])
-    // const roomsCol = collection( db, "rooms");
-    // const snapShot = getDocs(roomsCol);
-    console.log(db)
-
-    // const docRef = doc(db, "rooms");
-    // const docSnap = getDoc(docRef);
-
-    // if (docSnap.exists()) {
-    // console.log("Document data:", docSnap.data());
-    // } else {
-    // // doc.data() will be undefined in this case
-    // console.log("No such document!");
-    // }
+  const [channels, setChannels] = useState([])
+  useEffect(()=>{
+    async function fetchChannels() {
+      const snap = await getDocs(collection(db, "rooms"))
+      const channelsCollection = []
+      snap.forEach((doc) => {
+        channelsCollection.push({
+          id: doc.id,
+          name: doc.data().name
+        })
+      })
+      setChannels(channelsCollection)
+    }
 
-    useEffect(()=>{
-        const roomsCol = collection(db, 'rooms');
-        const roomsSnapShot = getDocs(roomsCol);
-        setChannels(
-            roomsSnapShot.getDoc.map((doc)=>({
-                id: doc.id,
-                name: doc.data().name
-            }))
-        )
-    },[])
-
-
-    return (
-        
-        <div className="sidebar">
-            <div className="sidebar_header">
-                <div className="sidebar_info">
-                    <h2>MigraCode Barcelona</h2>
-                    <h3>
-                        <FiberManualRecordIcon/>
-                        Ali raza Ashraf
-                    </h3>
-
-                </div>
-                <div className="sidebar_createicon">
-                    <CreateIcon/>
-                </div>
-                
-
-            </div>
```

Comment:
When working with other people is important to have a clean indentation. Work on your editor to makes sure you indentation is ok. Personally I prefer spaces over tabs. And with 2 space code looks compact and easier to read. 

But you could prefer tabs and 4 spaces. Stick to your choice. This code  have multiple lines of space. Looks disorganized. I know you're learning and this is the less important. But the sooner you get used to write nice code the better : )

Reply:
I will start working on it right away. Thank you so much, you put a lot of effort into helping me here and I will pay for it by completing this project asap.

Reply:
Happy to help :) 
Keep working, you're doing great 💪